rosetta-testing/src/test/java/com/regnosys/rosetta/validation/RosettaValidatorTest.xtend

```diff
@@ -1687,11 +1687,11 @@ class RosettaValidatorTest implements RosettaIssueCodes {
 				
 				Foo:
 					+ foo
-						[value "bar" path "baz" dateFormat "MMB/dd/yy"]
+						[value "bar" path "baz" dateFormat "MMb/dd/yy"]
 			}
 		'''.parseRosetta
 		model.assertError(ROSETTA_SYNONYM_BODY, null,
-			"Format must be a valid date/time format - Unknown pattern letter: B")
```

Comment:
The date format character `B` has been added to newer Java versions.

Reply:
Now that I have fixed the `--release` option, this doesn't need to change actually. I'll just leave this change though for future proofness if we ever upgrade to use `--release 17`.